onboard/catkin_ws/src/offboard_comms/Arduino Sketchbooks/ThrusterServoArduino/MultiplexedServo.cpp

```diff
@@ -2,15 +2,51 @@
 #include <Wire.h>
 #include "Adafruit_PWMServoDriver.h"
 #include "MultiplexedServo.h"
-#include "MultiplexedObject.h"
 
-MultiplexedServo::MultiplexedServo(Adafruit_PWMServoDriver *_multiplexer,int _num):MultiplexedObject(_multiplexer, _num){}
+MultiplexedServo::MultiplexedServo(Adafruit_PWMServoDriver *_multiplexer):multiplexer(_multiplexer){
```

Comment:
I would recommend making this just a default constructor (doesn't take any arguments), and then passing in the multiplexer value in a separate `initialise` function. Then you can avoid the heap allocation using `new` in `setup`, by just making `thrusters` and `servos` an array of `MultiplexedServo` instead of an array of `MultiplexedServo*`.